alphas/alpha_functions.py

```diff
@@ -29,3 +29,60 @@ def alpha_001(data: pd.DataFrame, ticker: str) -> pd.DataFrame:
     drop_signal_indices = data["actively_traded"].where(data["actively_traded"] == False).dropna().index
     raw_signal.loc[drop_signal_indices] = 0
     return raw_signal
+
+def alpha_021(data): # MARCUS COMPATIBLE
```

Comment:
add the alpha definitions as docstrings within the fxns and remove the comments haha

Reply:
also, add the following imports ar the top:

```
import os
import sys
from pathlib import Path

sys.path.append(os.getcwd())
sys.path.append(str(Path(os.getcwd()).parent.absolute()))

from alpha_utils import *
```